app/search_builders/local_authorities_search_builder.rb

```diff
@@ -3,7 +3,9 @@ class LocalAuthoritiesSearchBuilder < Hydra::SearchBuilder
 
   def only_models_for_local_authorities(solr_params)
     solr_params[:fq] ||= []
-    solr_params[:fq] << "has_model_ssim:(\"#{Person.to_class_uri}\" OR \"#{Group.to_class_uri}\" OR \"#{Organization.to_class_uri}\" OR \"#{Agent.to_class_uri}\")"
+    models = [Agent, Person, Group, Organization, Topic]
+    query_value = models.map {|model| "\"#{model.to_class_uri}\"" }.join(' OR ')
+    solr_params[:fq] << "has_model_ssim:(#{query_value})"
```

Comment:
You should write this using a term query. https://github.com/curationexperts/alexandria-v2/blob/import_marc/app/search_builders/search_builder.rb#L6